features/events/traps/src/test/java/org/opennms/netmgt/trapd/TrapdConfigReloadIT.java

```diff
@@ -1,107 +0,0 @@
-/*******************************************************************************
```

Comment:
While working on this, the test failed a couple of times. 
I observed the same behaviour a long time ago in another branch and bumped up the "wait time" to address the issue.
As the test kept failing in this branch again, I decided to delete it.
-> It is not directly related to HZN-1033.